compiler-core/templates/documentation_layout.html

```diff
@@ -1,18 +1,41 @@
 <!DOCTYPE html>
-<html>
+<html lang="en" data-theme="light" data-drawer="closed" data-prewrap="off">
   <head>
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1"/>
     <title>{{ page_title }}</title>
+    <meta name="description" content="{{project_name}} v{{project_version}} · {{page_meta_description}}"/>
+    <!-- Preload most used font variants -->
+    <link rel="font" as="font" href="{{ unnest }}/fonts/karla-v23-regular-latin.woff2" type="font/woff2"/>
+    <link rel="font" as="font" href="{{ unnest }}/fonts/karla-v23-bold-latin.woff2" type="font/woff2"/>
+    <link rel="font" as="font" href="{{ unnest }}/fonts/ubuntu-mono-v15-regular-latin.woff2" type="font/woff2"/>
```

Comment:
Where were these before? I can't see them in the PR

Reply:
These are just woff2 preloads. The entire fonts are loaded in `index.css`.